### ISSUE TITLE
Support for MLP batch normalization

### ISSUE THREAD

**Issue by rinuboney** [tensorflow/tensorflow#513]
any plans to support batch normalization for MLP soon?

**Comment by facadev** [comment]
Would be a helpful feature.

**Comment by vincentvanhoucke** [comment]
https://github.com/tensorflow/tensorflow/issues/804
